### PR TITLE
feat: added traceback echo if importerror occurs

### PR DESCRIPTION
**Pull request by IgnatovFedor**
Should help to solve issues like #1067 and #1189

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -12,18 +12,28 @@
...
+setup = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(setup)
 
 # -- Project information -----------------------------------------------------
 
 project = 'DeepPavlov'
-copyright = '2018, ' + deeppavlov.__author__
-author = deeppavlov.__author__
+copyright = f'2018, {setup.__author__}'
+author = setup.__author__
```
Comment (yoptar) [inline docs/conf.py:33]: This doesn't feel right...

File: setup.py [hunk 2]
```diff
@@ -14,11 +14,30 @@
...
+    meta_path = os.path.join(__location__, 'deeppavlov', '_meta.py')
+    with open(meta_path, 'w') as meta:
+        meta.writelines([
+            f'__version__ = {repr(__version__)}\n',
+            f'__author__ = {repr(__author__)}\n',
+            f'__description__ = {repr(__description__)}\n',
+            f'__keywords__ = {repr(__keywords__)}\n',
+            f'__license__ = {repr(__license__)}\n',
+            f'__email__ = {repr(__email__)}\n'
+        ])
```
Comment (yoptar) [inline setup.py:38]: We can do a `_meta.py` file and then do:
```python3
with open(meta_path) as meta:
    exec(meta)
```
to get all the variables without importing the `deeppavlov` package.
Would it be better?

File: docs/conf.py (hunk 1 above)
Comment (IgnatovFedor) [reply docs/conf.py:33]: Only change 2018 -> 2020?

Comment (yoptar) [reply docs/conf.py:33]: Importing `setup` in docs doesn't feel right for some reason (=

**Pull request merged**